src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -42,7 +42,8 @@ public Revolver() {
    * @param speed the voltage to apply to the motor as a percentage from -1 to 1
    */
   public void set(double speed) {
-    m_revolver.set(speed);
+    m_revolver.set(ControlMode.PercentOutput, speed);
+    m_setSpeed = speed;
```

Comment:
This is not necessary if you switch `m_revolver` to a `WPI_TalonFX`

Reply:
Resolved in commit 4b623d0